### PR TITLE
allow llm compress without convert

### PR DESCRIPTION
**Pull request by eaidova**
additional improvements:
* add opportunity provide tokenizer separately
* move export configs for reducing code duplication
* add shortcuts for cli args

### CODE REVIEW

File: llm_bench/python/convert.py [hunk 1]
```diff
@@ -48,13 +41,57 @@
...
     PYTORCH = 'pytorch'
     OPENVINO = 'openvino'
 
 
+def is_ov_model_provided(model_id, model_dir, precision, model_name="openvino_model.xml"):
+    model_dirs = []
+    if Path(model_id).is_dir():
+        model_dirs.append(Path(model_id))
+        model_dirs.append(Path(model_id) / precision)
+        model_dirs.append(Path(model_id) / 'dldt' / precision)
```
Comment (AlexKoff88) [inline llm_bench/python/convert.py:60]: A good practice is to introduce constants for folder names ('dldt', 'pytorch', etc.) and use them across the code. I guess you can do it in a separate PR.

**Review by AlexKoff88 — APPROVED** [review]

File: llm_bench/python/convert.py [hunk 2]
```diff
@@ -45,164 +37,88 @@
     from optimum.exporters.onnx.__main__ import _get_submodels_and_onnx_configs as _get_submodels_and_export_configs
 
 from transformers import AutoTokenizer, AutoConfig, AutoModelForCausalLM, AutoModelForSeq2SeqLM, AutoModel
-from utils.nncf_utils import COMPRESSION_OPTIONS, INT4_MODEL_CONFIGURATION, get_compressed_path
-from utils.model_utils import add_stateful_model_arguments
-from optimum.exporters.openvino.utils import flattenize_inputs
+from utils.nncf_utils import get_compressed_path COMPRESSION_OPTIONS, INT4_MODEL_CONFIGURATION, get_compressed_path
```
Comment (eaidova) [inline llm_bench/python/convert.py:40]:
```suggestion
from utils.nncf_utils import get_compressed_path, COMPRESSION_OPTIONS, INT4_MODEL_CONFIGURATION
```
